src/CardCatalog.js

```diff
@@ -122,6 +125,7 @@ const CardDemos = () => {
     <section className='demo-card-collection'>
       <Card image />
       <Card actions />
+      <Card actions shaped />
```

Comment:
Should we just apply `shaped` to the existing `<Card actions />` rather than duplicate it?

Reply:
I prefer keeping this separate because the above actions demos with default shape radius as per spec.